### PR TITLE
When sending error messages to shell, break them into chunks

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by aelred** [comment]
!rtg - all the random failures: https://jenkins.grakn.ai/job/unit-test-pull-request-grakn-titan/4033/#showFailuresLink

**Comment by aelred** [comment]
!rtg - cassandra timeouts (already logged)

**Comment by aelred** [comment]
!rtg - `CommitLogControllerTest.whenCommittingGraph_TaskManagerReceivesCountTask` @alexandraorth you lied to me

File: grakn-test/src/test/java/ai/grakn/test/graql/shell/GraqlShellIT.java [hunk 1]
```diff
@@ -413,6 +413,18 @@ public void testLargeQuery() throws Exception {
     }
 
     @Test
+    public void whenErrorIsLarge_UserStillSeesEntireErrorMessage() throws Exception {
```
Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/graql/shell/GraqlShellIT.java]: Another Graql Shell test to randomly fail I see :P

**Pull request merged**